scripts/zones/Waughroon_Shrine/mobs/Platoon_Scorpion.lua

```diff
@@ -39,10 +39,10 @@ entity.onMobSpawn = function(mob)
         -- Scorpions are bound for 10 secounds
         if math.random() <= 0.25 and skillID == 355 then
             mobArg:showText(mob,ID.text.SCORPION_IS_BOUND)
-            mobArg:addStatusEffect(xi.effect.BIND,1,0,10)
+            mobArg:addStatusEffect(xi.effect.BIND, 1, 0, 10)
 
         -- Scorpions can still move around, but will not auto attack
-        elseif math.random() <= 0.25 and skillID == 355 then
+        elseif math.random() <= 0.25 and skillID == 354 then
```

Comment:
@dallano I found this error from the lua extension, not completely sure this is what was needed to be done here.